paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/BillingDetailsHelpers.kt

```diff
@@ -96,4 +107,45 @@ internal object BillingDetailsHelpers {
             }
         else -> null
     }
+
+    internal suspend fun connectBillingDetailsFields(elementsFlow: Flow<List<FormElement>>) {
+        var phoneNumberElement: PhoneNumberElement? = null
+
+        elementsFlow.map { elementsList ->
+            elementsList
+                .filterIsInstance<SectionElement>()
+                .flatMap { it.fields }
+                .filterIsInstance<PhoneNumberElement>()
+                .firstOrNull()
+        }.collect {
+            phoneNumberElement = it
```

Comment:
Talked offline, ideally this would be a combine of these 2 flows, but can't figure out why that doesn't work.